ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -693,5 +693,15 @@ public void MultipleUnaryOperators(string formula, double expectedResult)
 
             Assert.AreEqual(expectedResult, res, XLHelper.Epsilon);
         }
+
+        [TestCase("RIGHT(\"2020\", 2) + 1", 21)]
+        [TestCase("LEFT(\"20.2020\", 6) + 1", 21.202)]
+        [TestCase("MID(\"2020.2020\", 3, 6) + 1", 21.202)]
```

Comment:
There is basically no difference between cases 2 and 3. I'd rather add something like '=2+3&4+5' to also test operators priority.

Reply:
Or '=0.1+0.2="0.3"'

Reply:
I added a few more test cases around the & operator. I had to change the direct cast to double because those formulas resolved to a string type.